src/wwwroot/services/settings.js

```diff
@@ -222,5 +223,14 @@
         url: url
       });
     }
+
+    function getCustomerDataByCuit(cuit) {
+      return $http({
+        actionDescription: 'action_getting_customer_data',
+        method: 'GET',
+        url: RELAY_CONFIG.baseUrl + '/customerTax',
```

Comment:
I'm not sure if the URL should be /consumerTax

Reply:
This will be changed, depend on the [CuitService](https://github.com/FromDoppler/cuit-service)

Reply:
I think that it is enough defined to apply the right URL, see https://github.com/FromDoppler/cuit-service/blob/master/CuitService/Controllers/TaxInfoController.cs#L18

What do you think about 
```suggestion
        url: `https://cuitservice.fromdoppler.com/taxinfo/by-cuit/${cuit}`
```

We can still define the domain, what do you think?

Reply:
And, for the fake service, I purpose `https://fakecuitservice.fromdoppler.net/taxinfo/by-cuit/${cuit}`